test/lit/passes/asyncify_pass-arg=asyncify-memory@memory.wast

```diff
@@ -0,0 +1,116 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; NOTE: This test was ported using port_passes_tests_to_lit.py and could be cleaned up.
+
+;; RUN: foreach %s %t wasm-opt --enable-multimemory --enable-mutable-globals --asyncify --pass-arg=asyncify-memory@memory -S -o - | filecheck %s
+
+(module
+(memory (export "memory") 1 1)
+(memory (export "unused") 1 1)
```

Comment:
```suggestion
  (memory $mem (export "memory") 1 1)
  (memory $ignore (export "unused") 1 1)
```
Indentation, and give them names. The names should make the test output easier to read. (Can update the output using `./scripts/update_lit_checks.py PATH-TO-TEST`)

Reply:
(I intentionally did not use the export names, so that the test verifies we use the export names to pick the memory.)